.github/actions/deploy-keptn-on-cluster/action.yml

```
@@ -18,14 +18,10 @@ inputs:
     required: false
     description: "Name of the kind cluster"
     default: "test-cluster"
-  scheduling-gates:
+  values_file:
```

to stay consistent with the naming
```suggestion
  values-file:
```